<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>378</width>
    <height>567</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>To-Do-List</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label1">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>-10</y>
      <width>261</width>
      <height>101</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>My To-Do List </string>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEdit_additem">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>90</y>
      <width>321</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QTextEdit {
    border: 1px solid gray;
    border-radius: 5px;
    padding: 4px;
}</string>
    </property>
   </widget>
   <widget class="QPushButton" name="add_task">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>150</y>
      <width>321</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
color: rgb(255, 255, 255);
background-color: #178884;
border-radius: 5px;
}
QPushButton:hover{ 
	background-color: rgb(0, 0, 0);
	color: rgb(255, 255, 255);
}</string>
    </property>
    <property name="text">
     <string>+ Add Task</string>
    </property>
   </widget>
   <widget class="QPushButton" name="clear_pushButton">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>450</y>
      <width>101</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton
{background-color: rgb(85, 170, 255);
color: rgb(255, 255, 255);
border-radius: 5px;

}
QPushButton:hover{ 
	background-color: rgb(0, 0, 0);
	color: rgb(255, 255, 255);
}</string>
    </property>
    <property name="text">
     <string>Clear all</string>
    </property>
   </widget>
   <widget class="QPushButton" name="delete_pushButton">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>450</y>
      <width>101</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
    background-color: rgb(255, 0, 0);
	color: rgb(255, 255, 255);
border-radius: 5px;

}
QPushButton:hover{
color: rgb(255, 255, 255);
background-color: rgb(0, 0, 0);
}
</string>
    </property>
    <property name="text">
     <string>Delete item</string>
    </property>
   </widget>
   <widget class="QListWidget" name="listWidget">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>210</y>
      <width>321</width>
      <height>221</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QListWidget {
    border: 1px solid gray;
    border-radius: 5px;
    padding: 4px;
}</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>378</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
